2020-04-10 11:10:31.755 -03:00 [INF] Starting Amplifier web host
2020-04-10 11:11:03.862 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 11:11:10.390 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger  
2020-04-10 11:11:10.553 -03:00 [INF] Request finished in 169.1247ms 301 
2020-04-10 11:11:10.770 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 11:11:10.969 -03:00 [INF] Request finished in 187.2166ms 200 text/html;charset=utf-8
2020-04-10 11:11:11.041 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/swagger-ui-standalone-preset.js  
2020-04-10 11:11:11.050 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/swagger-ui-bundle.js  
2020-04-10 11:11:11.129 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/swagger-ui.css  
2020-04-10 11:11:11.304 -03:00 [INF] The file /swagger-ui.css was not modified
2020-04-10 11:11:11.348 -03:00 [INF] Request finished in 219.5176ms 304 text/css
2020-04-10 11:11:11.755 -03:00 [INF] Sending file. Request path: '/swagger-ui-standalone-preset.js'. Physical path: 'N/A'
2020-04-10 11:11:11.761 -03:00 [INF] Request finished in 720.095ms 200 application/javascript
2020-04-10 11:11:12.160 -03:00 [INF] Sending file. Request path: '/swagger-ui-bundle.js'. Physical path: 'N/A'
2020-04-10 11:11:12.179 -03:00 [INF] Request finished in 1128.053ms 200 application/javascript
2020-04-10 11:11:12.586 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 11:11:12.828 -03:00 [INF] Request finished in 241.875ms 200 application/json;charset=utf-8
2020-04-10 11:19:03.066 -03:00 [INF] Starting Amplifier web host
2020-04-10 11:19:09.016 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 11:20:04.656 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 11:20:04.849 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 11:20:05.235 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 11:20:05.925 -03:00 [INF] Executing action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) - Validation state: "Valid"
2020-04-10 11:21:57.888 -03:00 [INF] Executed action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi), returned result Microsoft.AspNetCore.Mvc.OkResult in 111913.8739ms.
2020-04-10 11:21:57.940 -03:00 [INF] Executing HttpStatusCodeResult, setting HTTP status code 200
2020-04-10 11:21:57.960 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 112702.4731ms
2020-04-10 11:21:57.966 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 11:21:57.994 -03:00 [INF] Request finished in 113344.508ms 200 
2020-04-10 15:16:34.827 -03:00 [INF] Starting Amplifier web host
2020-04-10 15:16:48.110 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 15:17:01.902 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger  
2020-04-10 15:17:02.185 -03:00 [INF] Request finished in 293.7027ms 301 
2020-04-10 15:17:02.308 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 15:17:02.507 -03:00 [INF] Request finished in 199.6547ms 200 text/html;charset=utf-8
2020-04-10 15:17:02.593 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/swagger-ui-standalone-preset.js  
2020-04-10 15:17:02.594 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/swagger-ui-bundle.js  
2020-04-10 15:17:03.108 -03:00 [INF] Sending file. Request path: '/swagger-ui-standalone-preset.js'. Physical path: 'N/A'
2020-04-10 15:17:03.113 -03:00 [INF] Request finished in 520.4779ms 200 application/javascript
2020-04-10 15:17:03.128 -03:00 [INF] Sending file. Request path: '/swagger-ui-bundle.js'. Physical path: 'N/A'
2020-04-10 15:17:03.143 -03:00 [INF] Request finished in 548.5703ms 200 application/javascript
2020-04-10 15:17:03.511 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 15:17:04.102 -03:00 [INF] Request finished in 592.2451ms 200 application/json;charset=utf-8
2020-04-10 15:17:08.575 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 15:17:08.681 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 15:17:08.976 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 15:17:09.740 -03:00 [INF] Executing action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) - Validation state: "Valid"
2020-04-10 15:18:12.827 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 63829.195ms
2020-04-10 15:18:12.847 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 15:18:12.917 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: ExecuteReader requires an open and available Connection. The connection's current state is closed.
   at System.Data.SqlClient.SqlCommand.ValidateCommand(Boolean async, String method)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean asyncWrite, String method)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.SqlClient.SqlCommand.ExecuteReader()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 34
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 15:18:13.839 -03:00 [INF] Request finished in 65264.8997ms 500 text/plain
2020-04-10 15:18:49.424 -03:00 [INF] Starting Amplifier web host
2020-04-10 15:18:54.522 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 15:19:29.037 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 15:19:29.128 -03:00 [INF] Request finished in 96.4833ms 200 text/html;charset=utf-8
2020-04-10 15:19:29.413 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/favicon-32x32.png  
2020-04-10 15:19:29.422 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 15:19:29.526 -03:00 [INF] Sending file. Request path: '/favicon-32x32.png'. Physical path: 'N/A'
2020-04-10 15:19:29.529 -03:00 [INF] Request finished in 123.2165ms 200 image/png
2020-04-10 15:19:29.646 -03:00 [INF] Request finished in 224.4477ms 200 application/json;charset=utf-8
2020-04-10 15:19:41.697 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 15:19:41.754 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 15:19:41.863 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 15:19:42.115 -03:00 [INF] Executing action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) - Validation state: "Valid"
2020-04-10 15:19:48.714 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 6834.4637ms
2020-04-10 15:19:48.728 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 15:19:48.746 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 34
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 15:19:48.935 -03:00 [INF] Request finished in 7239.2571ms 500 text/plain
2020-04-10 15:23:11.015 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 15:23:11.019 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 15:23:11.025 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 15:23:11.165 -03:00 [INF] Executing action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) - Validation state: "Valid"
2020-04-10 15:23:17.218 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 6184.3095ms
2020-04-10 15:23:17.228 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 15:23:17.236 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 34
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 15:23:17.280 -03:00 [INF] Request finished in 6265.4852ms 500 text/plain
2020-04-10 15:28:23.893 -03:00 [INF] Starting Amplifier web host
2020-04-10 15:28:27.157 -03:00 [FTL] Host terminated unexpectedly
System.FormatException: Could not parse the JSON file.
 ---> System.Text.Json.JsonReaderException: 'B' is an invalid escapable character within a JSON string. The string should be correctly escaped. LineNumber: 2 | BytePositionInLine: 45.
   at System.Text.Json.ThrowHelper.ThrowJsonReaderException(Utf8JsonReader& json, ExceptionResource resource, Byte nextByte, ReadOnlySpan`1 bytes)
   at System.Text.Json.Utf8JsonReader.ConsumeStringAndValidate(ReadOnlySpan`1 data, Int32 idx)
   at System.Text.Json.Utf8JsonReader.ConsumeString()
   at System.Text.Json.Utf8JsonReader.ConsumeValue(Byte marker)
   at System.Text.Json.Utf8JsonReader.ReadSingleSegment()
   at System.Text.Json.Utf8JsonReader.Read()
   at System.Text.Json.JsonDocument.Parse(ReadOnlySpan`1 utf8JsonSpan, Utf8JsonReader reader, MetadataDb& database, StackRowStack& stack)
   at System.Text.Json.JsonDocument.Parse(ReadOnlyMemory`1 utf8Json, JsonReaderOptions readerOptions, Byte[] extraRentedBytes)
   at System.Text.Json.JsonDocument.Parse(ReadOnlyMemory`1 json, JsonDocumentOptions options)
   at System.Text.Json.JsonDocument.Parse(String json, JsonDocumentOptions options)
   at Microsoft.Extensions.Configuration.Json.JsonConfigurationFileParser.ParseStream(Stream input)
   at Microsoft.Extensions.Configuration.Json.JsonConfigurationProvider.Load(Stream stream)
   --- End of inner exception stack trace ---
   at Microsoft.Extensions.Configuration.Json.JsonConfigurationProvider.Load(Stream stream)
   at Microsoft.Extensions.Configuration.FileConfigurationProvider.Load(Boolean reload)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.Extensions.Configuration.FileConfigurationProvider.HandleException(ExceptionDispatchInfo info)
   at Microsoft.Extensions.Configuration.FileConfigurationProvider.Load(Boolean reload)
   at Microsoft.Extensions.Configuration.FileConfigurationProvider.Load()
   at Microsoft.Extensions.Configuration.ConfigurationRoot..ctor(IList`1 providers)
   at Microsoft.Extensions.Configuration.ConfigurationBuilder.Build()
   at Microsoft.AspNetCore.Hosting.WebHostBuilder.BuildCommonServices(AggregateException& hostingStartupErrors)
   at Microsoft.AspNetCore.Hosting.WebHostBuilder.Build()
   at BrainInBaseApi.Program.BuildWebHost(String[] args) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Program.cs:line 39
   at BrainInBaseApi.Program.Main(String[] args) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Program.cs:line 26
2020-04-10 15:29:35.556 -03:00 [INF] Starting Amplifier web host
2020-04-10 15:29:41.650 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 15:30:01.456 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 15:30:01.762 -03:00 [INF] Request finished in 328.6334ms 200 text/html;charset=utf-8
2020-04-10 15:30:02.313 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/favicon-32x32.png  
2020-04-10 15:30:02.394 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 15:30:02.444 -03:00 [INF] Sending file. Request path: '/favicon-32x32.png'. Physical path: 'N/A'
2020-04-10 15:30:02.447 -03:00 [INF] Request finished in 134.5613ms 200 image/png
2020-04-10 15:30:02.611 -03:00 [INF] Request finished in 217.2936ms 200 application/json;charset=utf-8
2020-04-10 15:30:06.455 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 15:30:06.530 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 15:30:06.748 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 15:30:07.029 -03:00 [INF] Executing action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) - Validation state: "Valid"
2020-04-10 15:30:49.599 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 42837.9573ms
2020-04-10 15:30:49.611 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 15:30:49.625 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 34
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 15:30:49.785 -03:00 [INF] Request finished in 43330.8326ms 500 text/plain
2020-04-10 15:35:46.967 -03:00 [INF] Starting Amplifier web host
2020-04-10 15:35:51.591 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 15:36:25.133 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 15:36:25.309 -03:00 [INF] Request finished in 182.9747ms 200 text/html;charset=utf-8
2020-04-10 15:36:25.827 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 15:36:25.827 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/favicon-32x32.png  
2020-04-10 15:36:25.876 -03:00 [INF] Sending file. Request path: '/favicon-32x32.png'. Physical path: 'N/A'
2020-04-10 15:36:25.879 -03:00 [INF] Request finished in 52.1929ms 200 image/png
2020-04-10 15:36:25.997 -03:00 [INF] Request finished in 169.3589ms 200 application/json;charset=utf-8
2020-04-10 15:36:30.869 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 15:36:30.914 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 15:36:31.019 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 15:36:31.200 -03:00 [INF] Executing action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) - Validation state: "Valid"
2020-04-10 15:36:37.400 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 6370.3688ms
2020-04-10 15:36:37.410 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 15:36:37.422 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 34
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 15:36:37.581 -03:00 [INF] Request finished in 6712.865ms 500 text/plain
2020-04-10 15:38:32.183 -03:00 [INF] Starting Amplifier web host
2020-04-10 15:38:36.219 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 15:38:47.434 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 15:38:47.591 -03:00 [INF] Request finished in 167.315ms 200 text/html;charset=utf-8
2020-04-10 15:38:48.156 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/favicon-32x32.png  
2020-04-10 15:38:48.160 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 15:38:48.234 -03:00 [INF] Sending file. Request path: '/favicon-32x32.png'. Physical path: 'N/A'
2020-04-10 15:38:48.238 -03:00 [INF] Request finished in 82.0393ms 200 image/png
2020-04-10 15:38:48.389 -03:00 [INF] Request finished in 228.44ms 200 application/json;charset=utf-8
2020-04-10 15:38:53.951 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 15:38:54.023 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 15:38:54.145 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 15:39:00.981 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 6816.726ms
2020-04-10 15:39:00.996 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 15:39:01.006 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 34
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 15:39:01.134 -03:00 [INF] Request finished in 7185.5008ms 500 text/plain
2020-04-10 15:41:43.531 -03:00 [INF] Starting Amplifier web host
2020-04-10 15:41:48.904 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 15:42:05.676 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 15:42:05.858 -03:00 [INF] Request finished in 188.95ms 200 text/html;charset=utf-8
2020-04-10 15:42:06.362 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/favicon-32x32.png  
2020-04-10 15:42:06.411 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 15:42:06.511 -03:00 [INF] Sending file. Request path: '/favicon-32x32.png'. Physical path: 'N/A'
2020-04-10 15:42:06.533 -03:00 [INF] Request finished in 171.6334ms 200 image/png
2020-04-10 15:42:06.652 -03:00 [INF] Request finished in 240.7154ms 200 application/json;charset=utf-8
2020-04-10 15:42:11.198 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 15:42:11.262 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 15:42:11.369 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 15:42:11.592 -03:00 [INF] Executing action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) - Validation state: "Valid"
2020-04-10 15:42:20.181 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 8794.109ms
2020-04-10 15:42:20.204 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 15:42:20.222 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 34
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 15:42:20.373 -03:00 [INF] Request finished in 9178.1696ms 500 text/plain
2020-04-10 15:43:40.885 -03:00 [INF] Starting Amplifier web host
2020-04-10 15:43:44.466 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 15:44:11.906 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 15:44:12.057 -03:00 [INF] Request finished in 158.7637ms 200 text/html;charset=utf-8
2020-04-10 15:44:12.411 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/favicon-32x32.png  
2020-04-10 15:44:12.416 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 15:44:12.455 -03:00 [INF] Sending file. Request path: '/favicon-32x32.png'. Physical path: 'N/A'
2020-04-10 15:44:12.471 -03:00 [INF] Request finished in 60.4154ms 200 image/png
2020-04-10 15:44:12.611 -03:00 [INF] Request finished in 194.3612ms 200 application/json;charset=utf-8
2020-04-10 15:44:16.969 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 15:44:17.008 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 15:44:17.145 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 15:45:44.630 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 87467.7827ms
2020-04-10 15:45:44.646 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 15:45:44.661 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 34
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 15:45:44.806 -03:00 [INF] Request finished in 87837.4097ms 500 text/plain
2020-04-10 15:48:55.663 -03:00 [INF] Starting Amplifier web host
2020-04-10 15:48:59.591 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 15:50:49.554 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 15:50:49.747 -03:00 [INF] Request finished in 197.0673ms 200 text/html;charset=utf-8
2020-04-10 15:50:50.271 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/favicon-32x32.png  
2020-04-10 15:50:50.274 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 15:50:50.337 -03:00 [INF] Sending file. Request path: '/favicon-32x32.png'. Physical path: 'N/A'
2020-04-10 15:50:50.341 -03:00 [INF] Request finished in 70.2571ms 200 image/png
2020-04-10 15:50:50.510 -03:00 [INF] Request finished in 235.2919ms 200 application/json;charset=utf-8
2020-04-10 15:50:54.179 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 15:50:54.246 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 15:50:54.355 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 15:51:20.074 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 25704.5223ms
2020-04-10 15:51:20.091 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 15:51:20.103 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 34
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 15:51:20.256 -03:00 [INF] Request finished in 26078.2024ms 500 text/plain
2020-04-10 15:52:58.700 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 15:52:58.720 -03:00 [INF] Request finished in 19.0635ms 200 text/html;charset=utf-8
2020-04-10 15:52:59.424 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 15:52:59.425 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/favicon-32x32.png  
2020-04-10 15:52:59.431 -03:00 [INF] Sending file. Request path: '/favicon-32x32.png'. Physical path: 'N/A'
2020-04-10 15:52:59.436 -03:00 [INF] Request finished in 12.5876ms 200 application/json;charset=utf-8
2020-04-10 15:52:59.446 -03:00 [INF] Request finished in 21.8093ms 200 image/png
2020-04-10 16:02:44.716 -03:00 [INF] Starting Amplifier web host
2020-04-10 16:02:50.408 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 16:03:33.777 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 16:03:33.938 -03:00 [INF] Request finished in 165.6879ms 200 text/html;charset=utf-8
2020-04-10 16:03:34.491 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/favicon-32x32.png  
2020-04-10 16:03:34.498 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 16:03:34.612 -03:00 [INF] Sending file. Request path: '/favicon-32x32.png'. Physical path: 'N/A'
2020-04-10 16:03:34.629 -03:00 [INF] Request finished in 138.1995ms 200 image/png
2020-04-10 16:03:34.728 -03:00 [INF] Request finished in 230.2761ms 200 application/json;charset=utf-8
2020-04-10 16:03:38.829 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 16:03:38.901 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 16:03:39.035 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 16:03:39.063 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 11.7572ms
2020-04-10 16:03:39.072 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 16:03:39.081 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'Microsoft.EntityFrameworkCore.DbContext' while attempting to activate 'BrainInBaseApi.Controllers.UsuarioController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 16:03:39.255 -03:00 [INF] Request finished in 426.4735ms 500 text/plain
2020-04-10 16:04:28.765 -03:00 [INF] Starting Amplifier web host
2020-04-10 16:04:32.552 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 16:04:37.044 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 16:04:37.140 -03:00 [INF] Request finished in 101.6011ms 200 text/html;charset=utf-8
2020-04-10 16:04:37.737 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/favicon-32x32.png  
2020-04-10 16:04:37.740 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 16:04:37.779 -03:00 [INF] Sending file. Request path: '/favicon-32x32.png'. Physical path: 'N/A'
2020-04-10 16:04:37.781 -03:00 [INF] Request finished in 44.0371ms 200 image/png
2020-04-10 16:04:37.871 -03:00 [INF] Request finished in 131.01ms 200 application/json;charset=utf-8
2020-04-10 16:04:41.465 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 16:04:41.534 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 16:04:41.630 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 16:04:41.652 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 9.843ms
2020-04-10 16:04:41.662 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 16:04:41.669 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'Microsoft.EntityFrameworkCore.DbContext' while attempting to activate 'BrainInBaseApi.Controllers.UsuarioController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 16:04:41.775 -03:00 [INF] Request finished in 309.5628ms 500 text/plain
2020-04-10 16:05:22.416 -03:00 [INF] Starting Amplifier web host
2020-04-10 16:05:27.204 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 16:05:27.928 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 16:05:28.027 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 16:05:28.148 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 16:05:43.644 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 15481.9836ms
2020-04-10 16:05:43.665 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 16:05:43.679 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.ArgumentException: Format of the initialization string does not conform to specification starting at index 0.
   at System.Data.Common.DbConnectionOptions.GetKeyValuePair(String connectionString, Int32 currentPosition, StringBuilder buffer, Boolean useOdbcRules, String& keyname, String& keyvalue)
   at System.Data.Common.DbConnectionOptions.ParseInternal(Dictionary`2 parsetable, String connectionString, Boolean buildChain, Dictionary`2 synonyms, Boolean firstKey)
   at System.Data.Common.DbConnectionOptions..ctor(String connectionString, Dictionary`2 synonyms)
   at System.Data.SqlClient.SqlConnectionString..ctor(String connectionString)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnectionOptions(String connectionString, DbConnectionOptions previous)
   at System.Data.ProviderBase.DbConnectionFactory.GetConnectionPoolGroup(DbConnectionPoolKey key, DbConnectionPoolGroupOptions poolOptions, DbConnectionOptions& userConnectionOptions)
   at System.Data.SqlClient.SqlConnection.ConnectionString_Set(DbConnectionPoolKey key)
   at System.Data.SqlClient.SqlConnection.set_ConnectionString(String value)
   at System.Data.SqlClient.SqlConnection..ctor(String connectionString)
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 31
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 16:05:43.834 -03:00 [INF] Request finished in 15917.9895ms 500 text/plain
2020-04-10 16:13:05.322 -03:00 [INF] Starting Amplifier web host
2020-04-10 16:13:10.792 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 16:13:47.575 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 16:13:47.697 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 16:13:47.841 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 16:13:48.091 -03:00 [INF] Executing action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) - Validation state: "Valid"
2020-04-10 16:13:48.411 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 555.5098ms
2020-04-10 16:13:48.420 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 16:13:48.429 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.ArgumentException: Format of the initialization string does not conform to specification starting at index 0.
   at System.Data.Common.DbConnectionOptions.GetKeyValuePair(String connectionString, Int32 currentPosition, StringBuilder buffer, Boolean useOdbcRules, String& keyname, String& keyvalue)
   at System.Data.Common.DbConnectionOptions.ParseInternal(Dictionary`2 parsetable, String connectionString, Boolean buildChain, Dictionary`2 synonyms, Boolean firstKey)
   at System.Data.Common.DbConnectionOptions..ctor(String connectionString, Dictionary`2 synonyms)
   at System.Data.SqlClient.SqlConnectionString..ctor(String connectionString)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnectionOptions(String connectionString, DbConnectionOptions previous)
   at System.Data.ProviderBase.DbConnectionFactory.GetConnectionPoolGroup(DbConnectionPoolKey key, DbConnectionPoolGroupOptions poolOptions, DbConnectionOptions& userConnectionOptions)
   at System.Data.SqlClient.SqlConnection.ConnectionString_Set(DbConnectionPoolKey key)
   at System.Data.SqlClient.SqlConnection.set_ConnectionString(String value)
   at System.Data.SqlClient.SqlConnection..ctor(String connectionString)
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 31
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 16:13:48.613 -03:00 [INF] Request finished in 1049.1123ms 500 text/plain
2020-04-10 16:14:15.103 -03:00 [INF] Starting Amplifier web host
2020-04-10 16:14:20.768 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 16:14:24.896 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 16:14:24.967 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 16:14:25.039 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 16:14:25.270 -03:00 [INF] Executing action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) - Validation state: "Valid"
2020-04-10 16:15:31.949 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 66897.5905ms
2020-04-10 16:15:31.955 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 16:15:31.963 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 35
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 16:15:32.135 -03:00 [INF] Request finished in 67241.5738ms 500 text/plain
2020-04-10 16:25:44.657 -03:00 [INF] Starting Amplifier web host
2020-04-10 16:25:48.961 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 16:26:04.026 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 16:26:04.148 -03:00 [INF] Request finished in 124.963ms 200 text/html;charset=utf-8
2020-04-10 16:26:04.744 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/favicon-32x32.png  
2020-04-10 16:26:04.792 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 16:26:04.840 -03:00 [INF] Sending file. Request path: '/favicon-32x32.png'. Physical path: 'N/A'
2020-04-10 16:26:04.842 -03:00 [INF] Request finished in 99.2807ms 200 image/png
2020-04-10 16:26:04.990 -03:00 [INF] Request finished in 198.334ms 200 application/json;charset=utf-8
2020-04-10 16:26:10.360 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 16:26:10.427 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 16:26:10.537 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 16:26:10.556 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 5.9704ms
2020-04-10 16:26:10.569 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 16:26:10.582 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'Microsoft.EntityFrameworkCore.DbContext' while attempting to activate 'BrainInBaseApi.Controllers.UsuarioController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 16:26:10.750 -03:00 [INF] Request finished in 390.7489ms 500 text/plain
2020-04-10 16:26:39.093 -03:00 [INF] Starting Amplifier web host
2020-04-10 16:26:42.726 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 16:26:44.369 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 16:26:44.443 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 16:26:44.538 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 16:26:56.316 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 11766.5993ms
2020-04-10 16:26:56.340 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 16:26:56.361 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 36
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 16:26:56.541 -03:00 [INF] Request finished in 12178.9696ms 500 text/plain
2020-04-10 16:28:00.459 -03:00 [INF] Starting Amplifier web host
2020-04-10 16:28:04.249 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 16:28:46.540 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 16:28:46.661 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 16:28:46.805 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 16:28:47.010 -03:00 [INF] Executing action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) - Validation state: "Valid"
2020-04-10 16:28:50.190 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 3360.5654ms
2020-04-10 16:28:50.212 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 16:28:50.237 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 36
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 16:28:50.394 -03:00 [INF] Request finished in 3858.4493ms 500 text/plain
2020-04-10 16:36:55.907 -03:00 [INF] Starting Amplifier web host
2020-04-10 16:37:00.554 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 16:37:16.594 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 16:37:16.695 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 16:37:16.836 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 16:37:32.642 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 15791.5997ms
2020-04-10 16:37:32.652 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 16:37:32.658 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 36
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 16:37:32.797 -03:00 [INF] Request finished in 16214.0686ms 500 text/plain
2020-04-10 17:58:42.539 -03:00 [INF] Starting Amplifier web host
2020-04-10 17:58:45.708 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 17:59:15.955 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 17:59:15.999 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 17:59:16.051 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 17:59:16.192 -03:00 [INF] Executing action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) - Validation state: "Valid"
2020-04-10 17:59:40.214 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 24154.8892ms
2020-04-10 17:59:40.224 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 17:59:40.233 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 34
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 17:59:40.344 -03:00 [INF] Request finished in 24392.9296ms 500 text/plain
2020-04-10 18:00:55.681 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 18:00:55.685 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 18:00:55.735 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 18:00:55.762 -03:00 [INF] Executing action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) - Validation state: "Valid"
2020-04-10 18:01:27.650 -03:00 [INF] Starting Amplifier web host
2020-04-10 18:01:30.352 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 18:02:09.076 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger  
2020-04-10 18:02:09.120 -03:00 [INF] Request finished in 48.763ms 301 
2020-04-10 18:02:09.154 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 18:02:09.196 -03:00 [INF] Request finished in 30.1116ms 200 text/html;charset=utf-8
2020-04-10 18:02:09.224 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/swagger-ui-standalone-preset.js  
2020-04-10 18:02:09.224 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/swagger-ui-bundle.js  
2020-04-10 18:02:09.225 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/swagger-ui.css  
2020-04-10 18:02:09.353 -03:00 [INF] Sending file. Request path: '/swagger-ui.css'. Physical path: 'N/A'
2020-04-10 18:02:09.366 -03:00 [INF] Sending file. Request path: '/swagger-ui-bundle.js'. Physical path: 'N/A'
2020-04-10 18:02:09.353 -03:00 [INF] Sending file. Request path: '/swagger-ui-standalone-preset.js'. Physical path: 'N/A'
2020-04-10 18:02:09.383 -03:00 [INF] Request finished in 158.3004ms 200 text/css
2020-04-10 18:02:09.392 -03:00 [INF] Request finished in 168.4212ms 200 application/javascript
2020-04-10 18:02:09.403 -03:00 [INF] Request finished in 179.133ms 200 application/javascript
2020-04-10 18:02:09.653 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 18:02:09.754 -03:00 [INF] Request finished in 100.7342ms 200 application/json;charset=utf-8
2020-04-10 18:02:19.296 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 18:02:19.323 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 18:02:19.379 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 18:02:19.483 -03:00 [INF] Executing action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) - Validation state: "Valid"
2020-04-10 18:02:54.413 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 35023.1811ms
2020-04-10 18:02:54.418 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 18:02:54.424 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 34
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 18:02:54.519 -03:00 [INF] Request finished in 35222.6586ms 500 text/plain
2020-04-10 18:13:39.108 -03:00 [INF] Starting Amplifier web host
2020-04-10 18:13:42.431 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 18:14:31.540 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 18:14:31.606 -03:00 [INF] Request finished in 70.4872ms 200 text/html;charset=utf-8
2020-04-10 18:14:31.780 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 18:14:31.884 -03:00 [INF] Request finished in 103.8982ms 200 application/json;charset=utf-8
2020-04-10 18:14:36.233 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 18:14:36.269 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 18:14:36.314 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 18:14:36.500 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 178.7297ms
2020-04-10 18:14:36.504 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 18:14:36.508 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 34
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 18:14:36.584 -03:00 [INF] Request finished in 351.4767ms 500 text/plain
2020-04-10 18:15:08.418 -03:00 [INF] Starting Amplifier web host
2020-04-10 18:15:11.914 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 18:15:17.346 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 18:15:17.403 -03:00 [INF] Request finished in 61.1324ms 200 text/html;charset=utf-8
2020-04-10 18:15:17.549 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 18:15:17.634 -03:00 [INF] Request finished in 84.8996ms 200 application/json;charset=utf-8
2020-04-10 18:15:21.346 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 18:15:21.397 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 18:15:21.467 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 18:15:21.562 -03:00 [INF] Executing action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) - Validation state: "Valid"
2020-04-10 18:15:21.719 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 246.4339ms
2020-04-10 18:15:21.722 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 18:15:21.728 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 34
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 18:15:21.798 -03:00 [INF] Request finished in 452.1191ms 500 text/plain
2020-04-10 18:16:15.425 -03:00 [INF] Starting Amplifier web host
2020-04-10 18:16:18.451 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 18:16:21.344 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 18:16:21.409 -03:00 [INF] Request finished in 66.8352ms 200 text/html;charset=utf-8
2020-04-10 18:16:21.547 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 18:16:21.635 -03:00 [INF] Request finished in 87.9973ms 200 application/json;charset=utf-8
2020-04-10 18:16:25.283 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 18:16:25.314 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 18:16:25.366 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 18:16:25.384 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 9.081ms
2020-04-10 18:16:25.391 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 18:16:25.396 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'Microsoft.EntityFrameworkCore.DbContext' while attempting to activate 'BrainInBaseApi.Controllers.UsuarioController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 18:16:25.468 -03:00 [INF] Request finished in 185.1601ms 500 text/plain
2020-04-10 18:17:22.695 -03:00 [INF] Starting Amplifier web host
2020-04-10 18:17:25.344 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 18:17:36.446 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 18:17:36.491 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 18:17:36.556 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 18:17:36.569 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 4.1695ms
2020-04-10 18:17:36.577 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 18:17:36.582 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'Microsoft.EntityFrameworkCore.DbContext' while attempting to activate 'BrainInBaseApi.Controllers.UsuarioController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 18:17:36.687 -03:00 [INF] Request finished in 243.3492ms 500 text/plain
2020-04-10 18:18:51.121 -03:00 [INF] Starting Amplifier web host
2020-04-10 18:18:54.386 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 18:18:59.784 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 18:18:59.846 -03:00 [INF] Request finished in 66.9594ms 200 text/html;charset=utf-8
2020-04-10 18:18:59.995 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 18:19:00.085 -03:00 [INF] Request finished in 89.3537ms 200 application/json;charset=utf-8
2020-04-10 18:19:03.721 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 18:19:03.751 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 18:19:03.811 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 18:19:03.825 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 4.2293ms
2020-04-10 18:19:03.830 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 18:19:03.836 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'Microsoft.EntityFrameworkCore.DbContext' while attempting to activate 'BrainInBaseApi.Controllers.UsuarioController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 18:19:03.908 -03:00 [INF] Request finished in 188.0146ms 500 text/plain
2020-04-10 18:19:47.962 -03:00 [INF] Starting Amplifier web host
2020-04-10 18:19:51.642 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 18:21:58.179 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 18:21:58.257 -03:00 [INF] Request finished in 78.5224ms 200 text/html;charset=utf-8
2020-04-10 18:21:58.450 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 18:21:58.557 -03:00 [INF] Request finished in 107.8481ms 200 application/json;charset=utf-8
2020-04-10 18:22:03.316 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 18:22:03.363 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 18:22:03.412 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 18:22:03.515 -03:00 [INF] Executing action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) - Validation state: "Valid"
2020-04-10 18:22:03.660 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 240.0001ms
2020-04-10 18:22:03.663 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 18:22:03.668 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 35
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 18:22:03.767 -03:00 [INF] Request finished in 451.2804ms 500 text/plain
2020-04-10 18:28:04.817 -03:00 [INF] Starting Amplifier web host
2020-04-10 18:28:08.052 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 18:28:20.963 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 18:28:21.021 -03:00 [INF] Request finished in 61.8365ms 200 text/html;charset=utf-8
2020-04-10 18:28:21.190 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 18:28:21.283 -03:00 [INF] Request finished in 92.3161ms 200 application/json;charset=utf-8
2020-04-10 18:28:25.865 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 18:28:25.892 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 18:28:25.943 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 18:28:26.062 -03:00 [INF] Executing action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) - Validation state: "Valid"
2020-04-10 18:28:26.235 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 285.4596ms
2020-04-10 18:28:26.241 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 18:28:26.246 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 35
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 18:28:26.335 -03:00 [INF] Request finished in 470.7709ms 500 text/plain
2020-04-10 20:55:00.124 -03:00 [INF] Starting Amplifier web host
2020-04-10 20:55:14.110 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 20:55:36.396 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/index.html  
2020-04-10 20:55:37.030 -03:00 [INF] Request finished in 656.1803ms 200 text/html;charset=utf-8
2020-04-10 20:55:38.063 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/swagger/v1/swagger.json  
2020-04-10 20:55:38.711 -03:00 [INF] Request finished in 648.3012ms 200 application/json;charset=utf-8
2020-04-10 20:55:47.112 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 20:55:47.240 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 20:55:47.722 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 20:55:48.122 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 356.6806ms
2020-04-10 20:55:48.138 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 20:55:48.188 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'Microsoft.EntityFrameworkCore.DbContext' while attempting to activate 'BrainInBaseApi.Controllers.UsuarioController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 20:55:48.732 -03:00 [INF] Request finished in 1620.3835ms 500 text/plain
2020-04-10 20:56:32.726 -03:00 [INF] Starting Amplifier web host
2020-04-10 20:56:36.300 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 20:56:48.868 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 20:56:49.003 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 20:56:49.134 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 20:56:50.051 -03:00 [INF] Executing action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) - Validation state: "Valid"
2020-04-10 20:57:33.080 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 43932.3323ms
2020-04-10 20:57:33.085 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 20:57:33.092 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 36
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 20:57:33.491 -03:00 [INF] Request finished in 44631.1299ms 500 text/plain
2020-04-10 21:03:29.144 -03:00 [INF] Starting Amplifier web host
2020-04-10 21:03:32.904 -03:00 [INF] User profile is available. Using 'C:\Users\Thiago_notAcer\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2020-04-10 21:03:39.777 -03:00 [INF] Request starting HTTP/1.1 GET http://localhost:2000/api/Usuario  
2020-04-10 21:03:39.874 -03:00 [INF] Executing endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 21:03:39.974 -03:00 [INF] Route matched with {action = "FindUsuario", controller = "Usuario"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] FindUsuario(BrainInBaseClass.Usuarios.Usuario) on controller BrainInBaseApi.Controllers.UsuarioController (BrainInBaseApi).
2020-04-10 21:03:40.200 -03:00 [INF] Executing action method BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) - Validation state: "Valid"
2020-04-10 21:04:00.450 -03:00 [INF] Executed action BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi) in 20467.0641ms
2020-04-10 21:04:00.457 -03:00 [INF] Executed endpoint 'BrainInBaseApi.Controllers.UsuarioController.FindUsuario (BrainInBaseApi)'
2020-04-10 21:04:00.464 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.InvalidOperationException: The ConnectionString property has not been initialized.
   at System.Data.SqlClient.SqlConnection.PermissionDemand()
   at System.Data.SqlClient.SqlConnectionFactory.PermissionDemand(DbConnection outerConnection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at BrainInBaseApi.Controllers.UsuarioController.FindUsuario(Usuario filtro) in D:\Projetos\BrainInBase\BrainInBaseApi\BrainInBase\Controllers\UsuarioController.cs:line 36
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-04-10 21:04:00.609 -03:00 [INF] Request finished in 20838.0409ms 500 text/plain
